macros.in

```diff
@@ -1021,6 +1021,7 @@ package or when debugging this package.\
   CFLAGS="${CFLAGS:-%optflags}" ; export CFLAGS ; \
   CXXFLAGS="${CXXFLAGS:-%optflags}" ; export CXXFLAGS ; \
   FFLAGS="${FFLAGS:-%optflags}" ; export FFLAGS ; \
+  LDFLAGS="${LDFLAGS:-%?ldflags}" ; export LDFLAGS; \
```

Comment:
But... we don't have `%ldflags`? I know this should initialize to an empty variable, but I don't know how that gets interpreted by autotools (or things pretending to offer autotools-like interfaces).

Reply:
Well, I would expect the `autotools` should be fine with empty `LDFLAGS`.